compiler/il/OMRILOps.hpp

```diff
@@ -37,8 +37,21 @@ namespace OMR
 
 enum VectorOperation
    {
+   vload,
+   vloadi,
+   vstore,
+   vstorei,
+   vnot,
+   vneg,
    vadd,
+   vsub,
+   vmul,
+   vdiv,
+   vand,
+   vor,
+   vxor,
    vfma,
+   vsplats,
```

Comment:
Yes, I saw your comment. I am about to open another PR which adds many more opcodes and they all are sorted. Is it OK if we delay it till next PR since otherwise I need to sort both this enum and all the macros in compiler/il/VectorOperations.enum twice?